pkg/rancher-desktop/assets/specs/command-api.yaml

```
@@ -325,6 +325,15 @@ components:
                     Use to disable flannel so you can install your own CNI.
                   x-rd-aliases:
                     - "flannel-enabled"
+            ingress:
+              type: object
+              properties:
+                localhostOnly:
+                  type: boolean
+                  x-rd-platforms: ['win32']
+                  x-rd-usage: >-
+                    Control whether K8s services are bound to 127.0.0.1 only or 0.0.0.0.
+                    Applies to NodePort and LoadBalancer types only.
```

#4116 has updated the usage message for many options and created a merge conflict for this file, so we need another rebase.

That PR also changed the style of the usage messages to be shorter phrases, starting in lowercase, and without a final dot. Since we have to touch this file anyways, I would suggest to update this usage message to match. E.g.

```suggestion
                  x-rd-usage: bind services to 127.0.0.1 instead of 0.0.0.0
```